ciao-cli/main.go

```diff
@@ -188,11 +193,11 @@ func sendHTTPRequestToken(method string, url string, values []queryValue, token
 	infof("Got HTTP response (status %s)\n", resp.Status)
 
 	if resp.StatusCode >= http.StatusBadRequest {
-		respBody, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			errorf("Could not read the HTTP response %s\n", err)
+		respBody, errBody := ioutil.ReadAll(resp.Body)
```

Comment:
@albertomurillo I see. A golint failure will prevent you from passing CI, so keep this change in please.